packages/ui/src/primitives/field.tsx

```diff
@@ -80,6 +85,21 @@ export const Hint = React.forwardRef(function Hint(
   );
 });
 
+export const Checkbox = React.forwardRef(function FieldCheckbox(
+  props: React.InputHTMLAttributes<HTMLInputElement>,
+  forwardedRef: React.ForwardedRef<HTMLInputElement>,
+) {
+  return (
+    <input
+      ref={forwardedRef}
+      type='checkbox'
+      data-field-checkbox
+      className={cx('accent-accent-9 mt-[0.1875em] size-3 cursor-pointer')}
```

Comment:
whats the reasoning to have margin top on the component here?

Reply:
This is so that when the `Label` goes beyond one line, it'll align nicely to the top line, rather than awkwardly in the middle